src/web/index.js

```diff
@@ -14,7 +14,11 @@ import { documentReady, getResourceLinks, getMeta } from './util';
 // used to create new `MessageContext` objects for a given `lang` with selected
 // builtin functions.
 function createContext(lang) {
-  return new MessageContext(lang);
+  // IE11 renders FSI and PDI as mojibakes so we turn them off.
+  const isIE11 = navigator.userAgent.indexOf('Trident') > -1;
```

Comment:
I'd change that to:
```
const isIE = !!document.documentMode;
```

The less UA-sniffing the better; UA can always change while no browser should ever implement the non-standard `document.documentMode`.